manage.py

```diff
@@ -92,7 +92,7 @@ def setup_general():
                 email=Config.ADMIN_EMAIL)
             db.session.add(user)
             db.session.commit()
-            print 'Added administrator {}'.format(user.full_name())
+            print(('Added administrator {}'.format(user.full_name())))
```

Comment:
Yes, 2to3 seems to like it that way, this PR was just a proof of concept no need to merge it, I'm pleased you'll consider Python 3 support for the project.